open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservice/assetcatalog/aspects/AssetCatalogExecutionAspect.java

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+package org.odpi.openmetadata.accessservice.assetcatalog.aspects;
+
+
+import org.aspectj.lang.ProceedingJoinPoint;
```

Comment:
The introduction of a new dependency requires developer documentation and a discussion amongst the maintainers.  This documentation needs to cover why the dependency is being introduced what is does and how to use it.